src/protocol/error.js

```diff
@@ -320,7 +320,7 @@ const errorCodes = [
   {
     type: 'CONCURRENT_TRANSACTIONS',
     code: 51,
-    retriable: false,
+    retriable: true,
```

Comment:
I'm planning to do a proper review tomorrow, but looking at the Kafka documentation this shouldn't be `true`.

https://kafka.apache.org/protocol.html#protocol_error_codes

Reply:
Right, saw your comment now. Interesting